### PR TITLE
Add RequestStore type definition

### PR DESCRIPTION
**Pull request by nogtk**
Add type definitions for [request_store](https://github.com/steveklabnik/request_store) gem.

I added only some methods written in [RequestStore's README](https://github.com/steveklabnik/request_store/blob/master/README.md).
This is the firest PR, and I'll add other method definitions as it is merged.

Could you review this PR?

### CODE REVIEW

File: gems/request_store/1.5/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# manifest.yaml describes dependencies which do not appear in the gemspec.
+# If this gem includes such dependencies, comment-out the following lines and
+# declare the dependencies.
+# If all dependencies appear in the gemspec, you should remove this file.
```
Comment (pocke) [inline gems/request_store/1.5/manifest.yaml:4]: Could you remove this file?

**Review by pocke — CHANGES_REQUESTED** [review]
Thanks for your PR! I just left one comment, could you please confirm that?

File: gems/request_store/1.5/manifest.yaml (hunk 1 above)
Comment (nogtk) [reply gems/request_store/1.5/manifest.yaml]: OK, I deleted in this commit https://github.com/ruby/gem_rbs_collection/pull/239/commits/ba20bf1598b33fad7191824e47c8bca6655595ff

**Comment by nogtk** [comment]
@pocke 
Thank you for reviewing!
I have sent re-Request review. Could you please confirm the fixed commit.

**Review by pocke — APPROVED** [review]
Thanks!

**Pull request merged**